Sources/Tools/MaxMain/plAgeDescInterface.cpp

```diff
@@ -197,7 +196,7 @@ BOOL plAgeDescInterface::DlgProc(HWND hDlg, UINT msg, WPARAM wParam, LPARAM lPar
         {
             HFONT origFont = (HFONT)SendMessage( hDlg, WM_GETFONT, 0, 0 );
             LOGFONT origInfo, newInfo;
-            GetObject( origFont, sizeof( origInfo ), &origInfo );
+            GetObjectA( origFont, sizeof( origInfo ), &origInfo );
```

Comment:
If you're explicitly using `GetObjectA`, you should also explicitly declare the variable type `LOGFONTA`